test/fork/tokenomics.js

```diff
@@ -0,0 +1,542 @@
+      expect(afterAssetsEthRate[asset], assertionErrorMsg('assetsEthRate', asset)).to.be.equal(value);
+    }
+
+    for (const [asset, value] of Object.entries(this.contractData.priceFeedOracle.before.assetsForEth)) {
+      expect(afterAssetsForEth[asset], assertionErrorMsg('assetsEthRate', asset)).to.be.equal(value);
+    }
+  });
+
+  it('Compares storage of upgrade LegacyGateway contract', async function () {
+    this.contractData.gateway.after._unused_nxmToken = await this.gateway._unused_nxmToken();
+    this.contractData.gateway.after._unused_tokenController = await this.gateway._unused_tokenController();
+    this.contractData.gateway.after._unused_quotationData = await this.gateway._unused_quotationData();
+    this.contractData.gateway.after._unused_claimsData = await this.gateway._unused_claimsData();
+    this.contractData.gateway.after._unused_claims = await this.gateway._unused_claims();
+    this.contractData.gateway.after._unused_pool = await this.gateway._unused_pool();
+    this.contractData.gateway.after._unused_memberRoles = await this.gateway._unused_memberRoles();
+    this.contractData.gateway.after.DAI = await this.gateway.DAI();
+    this.contractData.gateway.after._unused_incidents = await this.gateway._unused_incidents();
+    this.contractData.gateway.after._unused_coverMigrator = await this.gateway._unused_coverMigrator();
+
+    Object.entries(this.contractData.gateway.before).forEach(([key, value]) => {
+      expect(this.contractData.gateway.after[key], assertionErrorMsg(key)).to.be.equal(value);
+    });
+  });
+
+  it('Compares storage of upgrade Assessment contract', async function () {
+    this.contractData.assessment.after.assessment1 = await this.assessment.assessments(1);
+    this.contractData.assessment.after.stakeOf = await this.assessment.stakeOf(this.hugh);
+    this.contractData.assessment.after.votesOf = await this.assessment.votesOf(this.hugh, 1);
+    this.contractData.assessment.after.hasAlreadyVotedOn = await this.assessment.hasAlreadyVotedOn(this.hugh, 1);
+
+    Object.entries(this.contractData.assessment.before).forEach(([key, value]) => {
+      expect(this.contractData.assessment.after[key], assertionErrorMsg(key)).to.be.deep.equal(value);
+    });
+  });
+
+  it('Compares storage of upgrade IndividualClaims contract', async function () {
+    this.contractData.individualClaims.after.claim1 = await this.individualClaims.claims(1);
+    this.contractData.individualClaims.after.claimSubm1 = await this.individualClaims.lastClaimSubmissionOnCover(1);
+
+    Object.entries(this.contractData.individualClaims.before).forEach(([key, value]) => {
+      expect(this.contractData.individualClaims.after[key], assertionErrorMsg(key)).to.be.deep.equal(value);
+    });
+  });
+
+  it('Compares storage of upgrade YieldTokenIncidents contract', async function () {
+    await expect(this.yieldTokenIncidents.incidents(0)).to.be.reverted; // empty storage
```

Comment:
this always throws an error and from the trace it returns 0x0 so I'm assuming its empty?
```js
await this.yieldTokenIncidents.incidents(0);
```
incidents storage:
```solidity
Incident[] public override incidents;
```